Watchman.Discord/Areas/Commons/Models/Period.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Watchman.Discord.Areas.Commons.Models
+{
+    public enum Period
+    {
+        Hour,
+        Day,
+        Week,
+        Month
```

Comment:
Dodałbym jeszcze year, bo niektóre serwery mogą już istnieć od 2016, więc podsumowanie każdego roku byłoby przydatne.

Reply:
tylko że tutaj bazujemy na danych zebranych przez naszego bota, a nie danych dostarczanych przez discorda 

można później dodać opcje czytania chat i na tej podstawie aktualizowanie bazy, ale to jako osobne zadanie w przyszłości